.github/pull_request_template.md

```diff
@@ -1,11 +1,14 @@
-## Which issue does this PR close?
-
+## Issue to Close
 Closes #
 
-## Rationale for this change
+## Rationale
+
+
+## Changes
+
+
+## User-facing Changes
 
-## What changes are included in this PR?
 
-## Are there any user-facing changes?
+## Test Plan 
```

Comment:
This is non-intuitive than before.

Reply:
I would make the sub-header more concise because they will be included in the commit message. The `How does this change test` seems too verbose in the commit message.

Reply:
Let's merge this and trial run for a while